access_keys.go

```diff
@@ -0,0 +1,73 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"net/http"
+	"time"
+)
+
+type AccessKeysConfig struct {
+	KeyRotationIntervalDays int       `json:"key_rotation_interval_days"`
+	LastKeyRotationAt       time.Time `json:"last_key_rotation_at"`
+	DaysUntilNextRotation   int       `json:"days_until_next_rotation"`
+}
+
+type AccessKeysConfigUpdateRequest struct {
+	KeyRotationIntervalDays int `json:"key_rotation_interval_days"`
+}
+
+type accessKeysConfigResponse struct {
+	Response
+	Result AccessKeysConfig `json:"result"`
+}
+
+// AccessKeysConfig returns the Access Keys Configuration for an account.
+// API reference: https://api.cloudflare.com/#access-keys-configuration-get-access-keys-configuration
+func (api *API) AccessKeysConfig(ctx context.Context, accountID string) (AccessKeysConfig, error) {
+	return api.accessKeysConfig(ctx, accountID, AccountRouteRoot)
```

Comment:
we don't really need these intermediate methods; other Access resources have it where it's available at the account and zone level to reduce duplication.